script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_5_1.txt

```diff
@@ -5,7 +5,7 @@ But then why is the smell of blood so horribly real?
 "...A mis, take."
 A mistake  A mistake  A mistake  A mistake.
 A mistake  A mistake  A mistake  A mistake.
-Is it a mistake that I was the one who killed her.
+Is it a mistake that I was the one who killed her?
 Or is it a mistake that I wasn't the one who killed her.

```

Comment:
Should this actually be "Or would it have been a mistake if I hadn't killed her?" ?
> それとも、自分が殺していないという事が違うのか。

Reply:
Also on the line above, are we correct in emphasizing "Is it a mistake that **I was the one** that killed her" (implying... someone else would have)? Or should this just be "Is it a mistake that I killed her?"
> 自分が殺したというコトが違うのか。

Reply:
I think that might've been done because the "自分" is explicit in the sentence, when it might otherwise be implicit. I'm ok simplifying the sentence to that version though.